packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -213,6 +216,7 @@ class _ProductPageState extends State<ProductPage>
               upToDateProduct.website!.trim().isNotEmpty)
             _buildWebsiteWidget(upToDateProduct.website!.trim()),
         ],
+
```

Comment:
```suggestion
```

In flutter in general and more specifically here in this project you have to format the code.
Rather easy actually: basically it means (if possible) adding one comma before each closing parenthesis, and have your IDE format your code (with a command like ctrl-S).
Your PR cannot be accepted if the format is not OK.
More precisely, here you cannot have an empty line.